src/room.cpp

```diff
@@ -0,0 +1,59 @@
+// Copyright 2017 Tunnler Project
+// Licensed under GPLv2 or any later version
+// Refer to the license.txt file included.
+
+#include "tunnler/room.h"
+
+#include "tunnler/room_message_types.h"
+
+/// Maximum number of concurrent connections allowed to this room.
+static const uint32_t MaxConcurrentConnections = 10;
```

Comment:
In the long run this should be moved to the server creation / memberlimit. Temporarily it's fine